lib/eventstreams-util.js

```diff
@@ -30,6 +31,23 @@ function deserializer(kafkaMessage) {
     return kafkaMessage;
 }
 
+function redact(deserializer) {
+    return function(...args) {
+        kafkaMessage = deserializer(...args)
+        if (
+            kafkaMessage.message.meta.stream == "mediawiki.page_change.v1" && 
```

Comment:
I think you know, but you'll have to do this for every potential stream and schema out there.  mediawiki.revision-create, mediawiki.page-create, mediawiki.recentchange, etc. etc.